model/riscv_insts_aext.sail

```diff
@@ -29,23 +29,31 @@ function lrsc_width_str(width : word_width) -> string =
   }
 
 /**
- * RISC-V only appears to define LR / SC / AMOs for word and double, although
- * there seem to be encodings reserved for other widths.
+ * RISC-V A-extension defines LR / SC / AMOs for word and double
+ * RISC-V Zabha extension defines AMOs for byte and halfword
  */
-function amo_width_valid(size : word_width) -> bool = {
+function lrsc_width_valid(size : word_width) -> bool = {
   match(size) {
     WORD   => true,
     DOUBLE => sizeof(xlen) >= 64,
     _      => false
   }
 }
 
+function amo_width_valid(size : word_width) -> bool = {
+  match(size) {
```

Comment:
Nit: the style of the existing amo_width_valid was wrong, there should be a space after `match`, and the parens also aren't needed (though that mistake is a little more common in the model). As you're copying this code could you please fix it first so we don't propagate more instances of the same mistake?

Reply:
updated.